src/gp4prog/gp4prog.h

```diff
@@ -33,6 +33,12 @@
 
 #include <libusb-1.0/libusb.h>
 
+//Windows appears to define an ERROR macro in its headers.
+//Conflicts with ERROR enum defined in log.h.
+#if defined(_WIN32) && defined(ERROR)
+	#undef ERROR
+#endif
```

Comment:
That fixes part of the problem. However, `usb.cpp` will need `libusb`, and also depends on `log.h`, so the ERROR identifier still needs to be undefined.

Limits the scope of the "damage", but doesn't eliminate it.